src/index.js

```diff
@@ -0,0 +1,171 @@
+// import axios from 'axios';
+// "use strict";
+
+
+const changeTextColor = function(tempValue) {
+    
+    tempValueElement.classList.remove("red", "orange", "green", "teal", "yellow")
+    if (tempValue >= 80) {
+        tempValueElement.classList.add("red");
+    } else if ((69 < tempValue) && (tempValue < 80)) {
+        tempValueElement.classList.add("orange");
+    } else if ((59 < tempValue) && (tempValue < 70)) {
+        tempValueElement.classList.add("yellow");
+    } else if ((49 < tempValue) && (tempValue < 60)) {
+        tempValueElement.classList.add("green");
+    } else {
+        tempValueElement.classList.add("teal");
+    }
+}
+
+function increaseTemperature() {
+    const tempElement = document.getElementById('tempValue');
+    const currentTemp = parseInt(tempElement.textContent);
+    tempElement.textContent = currentTemp + 1;
+    changeTextColor(currentTemp)
+}
+
+function decreaseTemperature() {
+    const tempElement = document.getElementById('tempValue');
+    const currentTemp = parseInt(tempElement.textContent);
+    tempElement.textContent = currentTemp - 1;
+    changeTextColor(currentTemp)
+}
+// CHANGE SKY FUNCTION //
+
+const changeSky = function(event) {
+    const value = event.target.value;
+
+    skyElementSelector.classList.remove("cloudy", "sunny", "rainy", "snowy")
+    if (value === "Cloudy") {
+        skyElementSelector.classList.add("cloudy");
+        skyElementWithEmojis.innerText = "☁️☁️☁️☁️☁️"
+    } 
+    else if (value === "Sunny") {
+        skyElementSelector.classList.add("sunny");
+        skyElementWithEmojis.innerText = "☀️☀️☀️☀️☀️"
+    } 
+    else if (value === "Rainy") {
+        skyElementSelector.classList.add("rainy");
+        skyElementWithEmojis.innerText = "🌧️🌧️🌧️🌧️🌧️"
+    } 
+    else if (value === "Snowy") {
+        skyElementSelector.classList.add("snowy");
+        skyElementWithEmojis.innerText = "❄️❄️❄️❄️❄️"
+    } 
+}
+// END OF SKY CHANGE FUNCTION //
+
+
+// CHANGE THE CITY NAME IN HEADER // 
+
+const updateCityName = function(event) {
+    headerCityName.textContent = event.target.value 
+};
+// END OF CHANGING CITY NAME IN HEADER //
+
+
+// RESET THE HEADER BACK TO SEATTLE //
+
+const resetCityName = function (event) {
+    headerCityName.textContent = "Seattle";
+};
+// END OF HEADER RESET //
+
+
+// // API CALLS : //
+// const axios = require('axios');
+
+// const LOCATION_KEY = process.env['LOCATION_KEY'];
+// const WEATHERIQ_KEY = process.env['WEATHER_KEY'];
```

Comment:
The API keys were only necessary for the proxy server so we can remove this section of commented out code altogether.